openstack/glance/templates/migration-job.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       restartPolicy: OnFailure
       containers:
       - name: glance-migration
-        image: {{.Values.global.imageRegistry}}/{{.Values.global.imageNamespace}}/ubuntu-source-glance-api:{{ .Values.imageVersionGlanceApi | default .Values.imageVersion | default .Values.image_version }}
+        image: {{required ".Values.global.imageRegistry is missing" .Values.global.imageRegistry}}/{{.Values.global.imageNamespace}}/ubuntu-source-glance-api:{{required ".Values.imageVersion is missing" .Values.imageVersion | default .Values.imageVersionGlanceApi | default .Values.image_version }}
```

Comment:
regarding imageVersionGlanceApi: please use a consistent pattern, make it similar to the deployment